### PR TITLE
fix(antivirus): 2768 - limit antivirus ram usage & use unix socket

### PR DESCRIPTION
**Pull request by pitiscarf**
[2768](https://www.notion.so/jeveuxaider/Refacto-l-antivirus-882b9534e1e94e7da452ebaa0e133349)

It's not possible to use only clamscan (virus database is too long to load in engine)
clamd require 1,2Gb RAM only to load database, and more RAM is required to scan with clamscand

I prefer keep clamd outside of API.

Instead I did the following :
- Use unix socket over TCP (theorically faster especially in docker context)
- reduce RAM usage by modifying clamd config : (https://docs.clamav.net/manual/Installing/Docker.html#what-can-i-do-to-minimize-ram-usage)
- invoke freshclam at buildtime, to avoid downloading database at runtime (boost antivirus start)
- output clamd in /dev/stdout to get log in container context

### CODE REVIEW

**Comment by pierre-lebihan** [comment]
On peut tester l'HTTP 2 de scaleway aussi ?
C'es sensé être plus rapide aussi

Mais peut-être qu'il le faut a la fois sur l'api et l'antivirus pour que ça marche

Ticket ici : https://www.notion.so/jeveuxaider/SPIKE-Migration-vers-HTTP-2-sur-Scaleway-f0e34d45f9574001ba55ce831e8d7214?pvs=4

File: antivirus/Dockerfile [hunk 1]
```diff
@@ -19,22 +19,25 @@ ENV ENV production
 ENV PORT 8080
 ENV HOST 0.0.0.0
 
+RUN apk update && \
+    apk add clamav
```
Comment (pierre-lebihan) [inline antivirus/Dockerfile:23]: pourquoi virer libunrar ?

**Review by pierre-lebihan — APPROVED** [review]
Juste une question

File: antivirus/Dockerfile (hunk 1 above)
Comment (pitiscarf) [reply antivirus/Dockerfile:23]: c'est une dependance de clamav

**Pull request merged**